### ISSUE TITLE
clush of clush never ends

### ISSUE THREAD

**Issue by girardm** [cea-hpc/clustershell#302]
```
[root@madmin0 (ROME) ~]# clush -w gidc1 "/usr/bin/clush -w rome7-mn \"uname -n\""
gidc1: rome7-mn: rome7.bullx
```

The second clush (on gidc1) never ends
```
[root@gidc1 (ROME) ~]# pstree -ap 976
sshd,976 -D
  ├─sshd,1414
  │   └─clush,1417 /usr/bin/clush -w rome7-mn uname -n
  │       └─{clush},1422
```

**Comment by girardm** [comment]
Using clustershell-1.7-1 on both side
```
[root@madmin0 (ROME) ~]# rpm -qa|grep clustershell
clustershell-1.7-1.el7.20151120.noarch

[root@gidc1 (ROME) ~]# rpm -qa|grep clustershell
clustershell-1.7-1.el7.20151120.noarch
```

**Comment by thiell** [comment]
Hi @girardm and thanks for reporting this issue. It looks like the problem is related to the way the second clush is handling standard input.
As a workaround, you could add `--nostdin` to the second clush.

**Comment by girardm** [comment]
Hi @thiell and thanks for your quick answer. Your workaround gave me a another idea that work too.
'''
[root@madmin0 (ROME) ~]# clush -w gidc1 "/usr/bin/clush -w rome7-mn \"uname -n\" </dev/null"
gidc1: rome7-mn: rome7.bullx
[root@madmin0 (ROME) ~]#
'''

**Comment by thiell** [comment]
Ok I checked, this is a duplicate of #279
Fixed in 1.7.1 by 326702803746a178d0770c67c07e3b5fcf77920a
